src/main/js/components/dropdowns/utils.js

```diff
@@ -69,13 +49,21 @@ function generateDropdown(element, callback, immediate, options = {}) {
             });
           }
         },
-        onHide() {
-          const dropdowns = document.querySelectorAll("[data-tippy-root]");
-          const isMouseOverAnyDropdown = Array.from(dropdowns).some(
-            (dropdown) => dropdown.matches(":hover"),
-          );
+        onHide(instance) {
```

Comment:
Something in here is breaking the theme manager quick change under user profile.

<img width="293" height="519" alt="Image" src="https://github.com/user-attachments/assets/6157ef5f-33e6-4038-adfe-73fd76b21ce7" />

Clicking to change theme closes the jumplist

Reply:
Good spot, works in Safari annoyingly - fixed it now.